### PR TITLE
 update experiments.py: plenty of docs.

### PR DESCRIPTION
**Pull request by teytaud**
## Types of changes

- [x] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Motivation and Context / Related issue

Docs are now visible on the dashboard, so more doc ==> more cool.

## Checklist

- [x] The documentation is up-to-date with the changes I made.
- [x] I have read the [**CONTRIBUTING**](https://facebookresearch.github.io/nevergrad/contributing.html) document and completed the CLA (see [**CLA**](https://facebookresearch.github.io/nevergrad/contributing.html#contributor-license-agreement-cla)).
- [x] All tests passed, and additional code has been covered with new tests.

### CODE REVIEW

File: nevergrad/benchmark/experiments.py [hunk 1]
```diff
@@ -36,15 +36,19 @@
 # for black (since lists are way too long...):
 # fmt: off
 
+default_optims = None  # ["NGO10", "CMA", "Shiwa"]
```
Comment (jrapin) [inline nevergrad/benchmark/experiments.py:39]: why do you do that, you can do it better using the optimizer groups

Comment (jrapin) [reply nevergrad/benchmark/experiments.py:39]: optimizer groups will be way more sustainable, and allow shuffling, I would stronlgy advise to remove this

Comment (teytaud) [reply nevergrad/benchmark/experiments.py:39]: This does not do anything as long as its known. The only purpose is to replace or complete the list of optimizers when it's not None. So it's not the same purpose as optimizer group. The point is to run specifically a short list of optimizers on all benchmarks. I do not see any other option for this.

Ok for removing this feature if you don't like it.

**Comment by teytaud** [comment]
@jrapin  I modified; ok under this form ?

File: nevergrad/benchmark/experiments.py (hunk 1 above)
Comment (jrapin) [reply nevergrad/benchmark/experiments.py]: You can do the same think using the optimizer groups, just have one group that is always there (or always added) and contains nothing by default (and that you can change when needed)
you will also avoid duplicates this way

**Pull request merged**